shared/src/main/scala/drt/shared/UserPreferences.scala

```diff
@@ -0,0 +1,10 @@
+package drt.shared
+
+import upickle.default._
+
+case class UserPreferences(userSelectedPlanningTimePeriod: Option[Int],
```

Comment:
We want get rid of the option at this point, and just set the default value if it was null in the database. This is so that we don't spread the logic of default values out throughout the app and accidentally apply different defaults in different places, and also to keep the defaults easy to find.

I know you using this class to send the preferences to the server for updating. We can have a separate class for this purpose which does have the values as options